ui/src/services/utils.js

```diff
@@ -98,3 +173,11 @@ export const useRefreshEffect = (refreshAction, stopRefreshAction) => {
     };
   }, [dispatch, refreshAction, stopRefreshAction]);
 };
+
+export const sizeUnits = [
+  { label: 'ki', value: 'Ki', base: 10 ** 3 },
```

Comment:
The `i` units use power of two, not power of ten.

- `Ki` => 2**10 bytes
- `Mi` => 2**20 bytes
- `Gi` => 2**30 bytes
- `Ti` => 2**40 bytes
- `Pi` => 2**50 bytes